src/raymath.h

```diff
@@ -154,6 +154,17 @@ RMDEF float Lerp(float start, float end, float amount)
     return start + amount*(end - start);
 }
 
+// Normalize input value within input range
+RMDEF float Norm(float value, float start, float end)
```

Comment:
Is `Norm()` function name standard? I mean, do other math libraries use that naming convention? Why not `Normalize()` for example? I try to name functions with the best recognizable naming...

Reply:
It's like "lerp" they tend to use different name like "interpolate", "lin_inter" etc. So I'm not really sure about it. I'll change it to "Normalize"